cardano-config/src/Cardano/Config/Orphanage.hs

```diff
@@ -18,10 +20,22 @@ import qualified Data.Text as T
 
 import           Cardano.BM.Data.Tracer (TracingVerbosity(..))
 import qualified Cardano.Chain.Update as Update
+import           Cardano.Crypto.DSIGN.Ed448 (Ed448DSIGN)
+import           Cardano.Crypto.KES (SignKeyKES, SimpleKES, VerKeyKES)
 import qualified Ouroboros.Consensus.BlockchainTime as Consensus
 import           Ouroboros.Consensus.NodeId (NodeId(..), CoreNodeId (..))
+import           Ouroboros.Consensus.Shelley.Protocol.Crypto (TPraosStandardCrypto)
+import           Shelley.Spec.Ledger.Keys (VKeyES (..))
 
 
+deriving instance Generic (VKeyES TPraosStandardCrypto)
+deriving instance NFData (VKeyES TPraosStandardCrypto)
+
+deriving instance Generic Ed448DSIGN
```

Comment:
I don't think this is necessary:

```suggestion
```

Reply:
@Jimbo4350: My other suggested change seemingly doesn't work, but you should be clear to remove this.